Composer/packages/client/src/components/CreationFlow/v2/DefineConversation.tsx

```diff
@@ -0,0 +1,329 @@
+
+        if (imported) {
+          dataToSubmit.templateDir = templateDir;
+          dataToSubmit.eTag = eTag;
+          dataToSubmit.urlSuffix = urlSuffix;
+          dataToSubmit.alias = alias;
+          dataToSubmit.preserveRoot = true;
+
+          // create a notification to indicate import success
+          const notification = createNotification({
+            type: 'success',
+            title: '',
+            onRenderCardContent: ImportSuccessNotificationWrapper({
+              importedToExisting: false,
+            }),
+          });
+          addNotification(notification);
+        }
+      }
+
+      onSubmit(
+        {
+          ...dataToSubmit,
+        },
+        templateId || ''
+      );
+    },
+    [hasErrors, formData]
+  );
+
+  const onCurrentPathUpdateWrap = (newPath: string, storageId?: string) => {
+    onCurrentPathUpdate(newPath, storageId);
+    updateField('location', newPath);
+  };
+
+  useEffect(() => {
+    const location =
+      focusedStorageFolder !== null && Object.keys(focusedStorageFolder as Record<string, any>).length
+        ? Path.join(focusedStorageFolder.parent, focusedStorageFolder.name)
+        : '';
+    updateField('location', location);
+  }, [focusedStorageFolder]);
+
+  const locationSelectContent = useMemo(() => {
+    return (
+      <LocationSelectContent
+        createFolder={createFolder}
+        focusedStorageFolder={focusedStorageFolder}
+        operationMode={{ read: true, write: true }}
+        updateFolder={updateFolder}
+        onCurrentPathUpdate={onCurrentPathUpdateWrap}
+      />
+    );
+  }, [focusedStorageFolder]);
+  const dialogCopy = isImported ? DialogCreationCopy.IMPORT_BOT_PROJECT : DialogCreationCopy.DEFINE_BOT_PROJECT;
+
+  return (
+    <Fragment>
+      <DialogWrapper isOpen {...dialogCopy} dialogType={DialogTypes.CreateFlow} onDismiss={onDismiss}>
+        <form onSubmit={handleSubmit}>
+          <input style={{ display: 'none' }} type="submit" />
+          <Stack horizontal styles={stackinput} tokens={{ childrenGap: '2rem' }}>
+            <StackItem grow={0} styles={halfstack}>
+              <TextField
+                autoFocus
+                required
+                data-testid="NewDialogName"
+                errorMessage={formErrors.name}
+                label={formatMessage('Name')}
+                styles={name}
+                value={formData.name}
+                onChange={(_e, val) => updateField('name', val)}
+              />
+            </StackItem>
+            <StackItem grow={0} styles={halfstack}>
+              <TextField
+                multiline
+                label={formatMessage('Description')}
+                resizable={false}
+                styles={description}
+                value={formData.description}
+                onChange={(_e, val) => updateField('description', val)}
+              />
+            </StackItem>
```

Comment:
There are some fields missing from this --

we need to also collect the webapp/functions runtime choice here, as well as the primary language.  I believe there is a design for the language field in the figma files

Reply:
Okay I have added those fields and nothing is done with the selection as of this PR. We can take the selections and apply them to the runtime once the runtime is ready for that, I have created a task to track that.